src/axom/sidre/docs/sphinx/file_io.rst

```diff
@@ -47,10 +47,26 @@ which omits error-checking and recovery for brevity and clarity.
    :end-before: _serial_io_save_end
    :language: C++
 
-The ``loadExternalData()`` method is used to read "external" data from an HDF5 
+The ``loadExternalData()`` method is used to read external data from an HDF5 
 file created with the ``sidre_hdf5`` protocol.  This is data referred to by 
 *external* views. Such views refer to data that is not stored in Sidre buffers 
-but in an external data allocation.
+but in an external data allocation.  ``loadExternalData()`` enables the
+loading of a ``DataStore`` with external data that preserves the state that
+was originally saved.  This requires a three step process.
+
+.. literalinclude:: ../../tests/sidre_external.cpp
+   :start-after: _external_save_load_start
+   :end-before: _external_save_load_end
+   :language: C++
+
+The first step is to call ``load()`` as before.  This will load the entire
+Sidre hierarchy stored in the file, including all ``View`` descriptions,
+but it will *not* allocate storage or load data values for the views with
+external data.  Second, the calling code must provide each external ``View``
+a pointer to valid storage that will hold the external data, using
+``setExternalDataPtr()``.  Finally, ``loadExternalData()`` is called,
```

Comment:
Should a comment be added about describing the external data?

Reply:
There is no step needed to describe the external data when loading, as the first `load()` call restores the descriptions that were saved.  I mention that here, but I could try to change the wording?

Reply:
I must have missed it. It may be good to emphasize the point in bold or similar so readers catch it.